lib/local-links-manager/import/local_authorities_url_importer.rb

```diff
@@ -0,0 +1,56 @@
+require 'csv'
+
+module LocalLinksManager
+  module Import
+    class LocalAuthoritiesUrlImporter
```

Comment:
Is it worth adding "URL" to the inflections file, as it is generally understood to be ALL CAPS? Only the first letter in capitals looked weird to me when I first saw this class name, and we have prior art for it in [Transition](https://github.com/alphagov/transition/blob/master/config/initializers/inflections.rb#L23).